velox/docs/functions/spark/string.rst

```diff
@@ -164,6 +164,16 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT rtrim('kr', 'spark'); -- "spa"
 
+.. spark:function:: space(count) -> varchar
+
+    Returns a string consisting of `count` spaces. ::
```

Comment:
Nit: I note \`` are used to surround arg name instead of using one at other places. Maybe, better to keep consistent.